packages/office-ui-fabric-react/src/components/Panel/examples/Panel.Controlled.Example.tsx

```diff
@@ -15,7 +15,7 @@ export class PanelControlledExample extends React.Component<{}, IPanelMediumExam
     return (
       <div>
         <DefaultButton secondaryText="Opens the Sample Panel" onClick={this._showPanel} text="Open Panel" />
-        <Panel isOpen={this.state.showPanel} type={PanelType.medium} headerText="Controlled Panel">
+        <Panel isOpen={this.state.showPanel} closeButtonAriaLabel="Close" type={PanelType.medium} headerText="Controlled Panel">
```

Comment:
Looks like this is fixing Bug 1 in the Panel A11y pass - it'd make it more clear to add this PR # next to each of the bugs in that list and cross them out so which ones get fixed by why PR is obvious. Having a list of those bug #s this PR addresses would be good too (in this PR's description).